index.js

```diff
@@ -12,6 +12,10 @@ app.get('/', (req, res) => {
   res.send('Hello World');
 });
 
+app.get('/info', (request, response) => {
+  response.send('name: OpenMRS-express | version: 0.1.0');
```

Comment:
It should be a JSON format and need to be taken form the pakage.json. 
Read the task contents carefully :-)

Reply:
ok sure